kits/changegraph-release-intelligence/.env.example

```
@@ -0,0 +1,6 @@
+﻿LAMATIC_API_KEY=your_lamatic_api_key
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)